src/AutomationTests/AutomationIntegrationTests.cs

```diff
@@ -132,7 +132,7 @@ public void Scan_Integration_WildlifeManager_InvalidRoot(bool sync)
             });
         }
 
-        // [DataTestMethod]
+        // [TestMethod]
```

Comment:
```suggestion
        // [DataTestMethod]
```

Reply:
@codeofdusk Why? Ideally when the test is enabled back, you will not want it to use DataTestMethod still.

Reply:
Note that once MSTest analyzers are enabled in the repo, you should actually start getting a warning here because it's a public method missing the attribute and you'll need to uncomment it soon and use `[Ignore]`